src/azure-cli/azure/cli/command_modules/vm/_actions.py

```diff
@@ -89,25 +89,27 @@ def load_images_from_aliases_doc(cli_ctx, publisher=None, offer=None, sku=None):
     import requests
     from azure.cli.core.cloud import CloudEndpointNotSetException
     from azure.cli.core.util import should_disable_connection_verify
+    from azure.cli.command_modules.vm._alias import alias_json
     try:
         target_url = cli_ctx.cloud.endpoints.vm_image_alias_doc
     except CloudEndpointNotSetException:
-        raise CLIError("'endpoint_vm_image_alias_doc' isn't configured. Please invoke 'az cloud update' to configure "
-                       "it or use '--all' to retrieve images from server")
-    # under hack mode(say through proxies with unsigned cert), opt out the cert verification
-    from azure.cli.command_modules.vm._alias import alias_json
-    try:
-        response = requests.get(target_url, verify=(not should_disable_connection_verify()))
-        if response.status_code == 200:
-            dic = json.loads(response.content.decode())
-        else:
-            logger.warning("Failed to retrieve image alias doc '%s'. Error: '%s'. Use local copy instead.",
-                           target_url, response)
-            dic = json.loads(alias_json)
-    except requests.exceptions.ConnectionError:
-        logger.warning("Failed to retrieve image alias doc '%s'. Error: 'ConnectionError'. Use local copy instead.",
-                       target_url)
+        logger.warning("'endpoint_vm_image_alias_doc' isn't configured. Please invoke 'az cloud update' to configure "
```

Comment:
curious why it's warning now, prevoiusly is exception

Reply:
2 reasons:
1. A local copy is introduced. It should not halt. It should read local copy when online image alias is not configured or not available.
2. Even if image alias endpoint is not configured, it should not block creating vm from a custom image.

Reply:
ok, so what's image alias endpoint used for?

Reply:
It is an URL where the image alias file locates. When using `vm create --image centos`, centos is an alias, it needs to be transformed to a standard form, which consists of publisher, offer, sku, version.